corehq/apps/userreports/reports/filters/values.py

```diff
@@ -229,6 +229,12 @@ def _is_empty(self):
         operator = self.value.get('operator') or '='
         return self._has_empty_value() and operator == '='
 
+    def _is_exists(self):
```

Comment:
does "is exists" mean the same as "exists"?

Reply:
i think of it as meaning "does this look like an "exists" filter?" which is a little different. it also matches the `_is_xx` naming convention of the rest of the class